apis/v1beta1/shared_types.go

```diff
@@ -288,6 +288,29 @@ const (
 	// This reason is used with the "ResolvedRefs" condition when one of the
 	// Route's rules has a reference to a resource that does not exist.
 	RouteReasonBackendNotFound RouteConditionReason = "BackendNotFound"
+
+	// This condition indicates whether configuration present in a Route is
+	// partially invalid. This may occur in the form of a combination of
+	// incompatible filters present in an individual route rule that an
+	// implementation wishes to call out. Controllers should only add this
+	// condition with status true when such an error is present.
```

Comment:
I kinda wanted to add something to the effect of "use this when Accepted=true but something is still wrong", lmk if people think that would be useful